internal/logic_handlers/organizationlogic/service.go

```diff
@@ -0,0 +1,179 @@
+package organizationlogic
+
+import (
+	"context"
+	"fmt"
+	"github.com/dembygenesis/local.tools/internal/model"
+	"github.com/dembygenesis/local.tools/internal/persistence"
+	"github.com/dembygenesis/local.tools/internal/sysconsts"
+	"github.com/dembygenesis/local.tools/internal/utilities/errs"
+	"github.com/dembygenesis/local.tools/internal/utilities/validationutils"
+	"github.com/sirupsen/logrus"
+	"net/http"
+	"strings"
+)
+
+type Config struct {
+	TxProvider persistence.TransactionProvider `json:"tx_provider" validate:"required"`
+	Logger     *logrus.Entry                   `json:"Logger" validate:"required"`
+	Persistor  persistor                       `json:"Persistor" validate:"required"`
+}
+
+func (i *Config) Validate() error {
+	return validationutils.Validate(i)
+}
+
+type Service struct {
+	cfg *Config
+}
+
+func New(cfg *Config) (*Service, error) {
+	if err := cfg.Validate(); err != nil {
+		return nil, fmt.Errorf("validate: %w", err)
+	}
+	return &Service{cfg}, nil
+}
+
+func (i *Service) AddOrganization(ctx context.Context, params *model.CreateOrganization) (*model.Organization, error) {
```

Comment:
I think we should just name this CreateOrganization because your model is *model.CreateOrganization

Reply:
Ah yes. On it.